Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -283,10 +283,64 @@ CFTypeID CGPathGetTypeID() {
     return __CGPath::GetTypeID();
 }
 
+// A helper for determining the number of points per path element type
+static inline int CGPathGetExpectedPointCountForType(CGPathElementType type) {
```

Comment:
I'd prefer if it's unsigned. I don't think the ptrdiff_t would be any more clear.

---
In reply to: [97700376](https://github.com/Microsoft/WinObjC/pull/1813#discussion_r97700376) [](ancestors = 97700376)